packages/@aws-cdk/aws-stepfunctions-tasks/test/apigateway/integ.invoke.ts

```diff
@@ -0,0 +1,48 @@
+import * as apigateway from '@aws-cdk/aws-apigateway';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { ApiGatewayInvoke, AuthType, HttpMethod } from '../../lib';
+
+/*
+ * Stack verification steps:
+ * * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn> : should return execution arn
+ * * aws stepfunctions describe-execution --execution-arn <exection-arn generated before> : should return status as SUCCEEDED and a query-execution-id
+ * * aws apigateway test-invoke-method --rest-api-id <value> --resource-id <value> --http-method <value>: should return the same response as the state machine
```

Comment:
should the rest-api-id also be an output? same question for all the inputs here. steps should include where they retrieved from. you might need to add outputs to the stack if they're not easily available.

Reply:
Could you clarify what you mean by this? I am not sure how to get the rest-api-id.